### PR TITLE
Combined landing page

### LINKED ISSUE #2: Combine landing page into this repo
Maintain the structure...

### PR DESCRIPTION
**Pull request by EshwariK**
fixes #2

### CODE REVIEW

File: public/assets/img/favicon/apple-touch-icon.png
Comment (visionarc-chemnitz) [inline public/assets/img/favicon/apple-touch-icon.png:1]: keep only those logo's which are actually used and remove others.

File: public/assets/img/mission.jpg
Comment (visionarc-chemnitz) [inline public/assets/img/mission.jpg:1]: Update these images with something good!!

File: public/assets/img/purpose.jpg
Comment (visionarc-chemnitz) [inline public/assets/img/purpose.jpg:1]: place those images that will be used in about us section under public/assets/img/aboutus

File: src/lib/prisma.ts
Comment (visionarc-chemnitz) [inline src/lib/prisma.ts:1]: was this necessary for liniting?  if yes can you update the eslintrc.json file for the same.

**Review by visionarc-chemnitz — CHANGES_REQUESTED** [review]
Some minor updates are required.

File: public/assets/img/favicon/apple-touch-icon.png
Comment (EshwariK) [reply public/assets/img/favicon/apple-touch-icon.png:1]: We may need them in future, so keeping for now.

**Review by OmkarMirgal — APPROVED** [review]
Only unused logo are left to removed.Can be done at later commits. For now all good!👍🏻

**Review by visionarc-chemnitz — APPROVED** [review]

**Pull request merged**